# Problem Statement 1:
Blood glucose levels for obese patients have a mean of 100 with a standard deviation of 15. 
A researcher thinks that a diet high in raw cornstarch will have a positive effect on
blood glucose levels. A sample of 36 patients who have tried the raw cornstarch diet
have a mean glucose level of 108. Test the hypothesis that the raw cornstarch had an
effect or not.

In [3]:
from scipy.stats import norm
import numpy as np

pop_mean = 100
stddev = 15
n = 36
sample_mean = 108
HT = stddev/n **0.5
Z = (sample_mean - pop_mean)/HT
print(f"Zscore is :{Z}")
alpha = 0.05
print(f"critical region is {norm.ppf(alpha/2)},{-norm.ppf(alpha/2)}")

Zscore is :3.2
critical region is -1.9599639845400545,1.9599639845400545


Hence we reject null hypothesis

# Problem Statement 2:
    
In one state, 52% of the voters are Republicans, and 48% are Democrats. In a second
state, 47% of the voters are Republicans, and 53% are Democrats. Suppose a simple
random sample of 100 voters are surveyed from each state.

What is the probability that the survey will show a greater percentage of Republican
voters in the second state than in the first state?

In [11]:
import math
n =100
p1 = 0.52
d1 = 0.48
p2 = 0.47
d2 = 0.53
mu = p1-p2
SD = math.sqrt(((p1*d1)/n) + ((p2*d2)/n))
#p1 - p2 < 0
x = 0
Z = (x - mu)/SD
print("standar deviation:",SD)
print("mean:",mu)
print("Z score :",Z)


standar deviation: 0.07061869440877536
mean: 0.050000000000000044
Z score : -0.7080278164104213


Hence from the z table we conclude that the probability that the survey will show a greater percentage of Republican
voters in the second state than in the first state is 0.24

# Problem Statement 3:
    
You take the SAT and score 1100. The mean score for the SAT is 1026 and the standard
deviation is 209. How well did you score on the test compared to the average test taker?

In [13]:
x = 1100
mean = 1026
stdev = 209
z = (x-mean)/stdev
z

0.35406698564593303

It shows that score is 0.35 standard deviations above the mean